dspace-spring-rest/src/main/java/org/dspace/app/rest/BitstreamContentRestController.java

```diff
@@ -9,63 +9,142 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.sql.SQLException;
 import java.util.UUID;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
 
+import org.apache.http.auth.AUTH;
+import org.apache.log4j.Logger;
 import org.dspace.app.rest.model.BitstreamRest;
-import org.dspace.app.rest.repository.BitstreamRestRepository;
-import org.dspace.core.Utils;
+import org.dspace.app.rest.utils.ContextUtil;
+import org.dspace.app.rest.utils.MultipartFileSender;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.content.Bitstream;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.core.Constants;
+import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.EventService;
+import org.dspace.usage.UsageEvent;
+import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.hateoas.Link;
-import org.springframework.hateoas.ResourceSupport;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 /**
  * This is a specialized controller to provide access to the bitstream binary content
- * 
+ *
  * @author Andrea Bollini (andrea.bollini at 4science.it)
+ * @author Atmire NV (info at atmire dot com)
```

Comment:
While we haven't really "policed" `@author` tags closely in recent years, I feel we should likely keep them pointing at individuals (who want credit for creating/refactoring specific classes) instead of organizations. I'm a bit worried that labeling classes with organizational contact info is a bit misleading, as it no longer refers to the specific creator of the class, and also could become an advertising gimmick/battle within our codebase (over which company gets more credit). The only company that "owns" the code is DuraSpace, and we just own the copyright in order to ensure the code remains open source, etc.

So, personally, I feel we should avoid this sort of change in our codebase. Either leave off the `@author` altogether, or reference the individual(s) who created / refactored the class.

However, if others disagree strongly here, I'll gladly bring this to discussion within the Committers group and/or Steering Group, so that we can formalize a policy on `@author` tags

Reply:
I only noted this in the first file I found with this company `@author` tag. There are a few others

Reply:
@tomdesair : Thanks for pointing out the Lyncode `@author` tags. I suspect those slipped in during the massive OAI refactor (i.e. creation of XOAI) many years back. It's simply hard to police this sort of thing. Had I noticed these at the time, I also would have questioned them.  And, I'd rather remove them then add in more.

FWIW, other major projects (Apache Foundation and Gradle are two, see this [blog post for example](http://vojtechruzicka.com/stop-using-javadoc-author-tag/)) do not allow any `@author` tags whatsoever. Their policy seem to be they quickly get out of date or are misleading, and simply cause clutter.  While I understand your reasoning, I think individuals should be attributed via *commits* or via name, and not via a blanket generic statement.   As-is, I think "Atmire NV (info at atmire dot com)" gives us no information about who actually worked on this code at Atmire, so there's no way to attribute those individuals in our [Contributor list](https://wiki.duraspace.org/display/DSPACE/DSpaceContributors), or even (eventually) consider those individuals for Committership